### PR TITLE
GitHub actions deploy example to gh pages

### PR DESCRIPTION
**Pull request by chepo92**
Adds action to build example git github actions to ease the start of new site from the example

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - main  # Set a branch to deploy
```
Comment (ryanfox1985) [inline .github/workflows/main.yml:6]: could you replace for master?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
```
Comment (ryanfox1985) [inline .github/workflows/main.yml:10]: perhaps use 20.X LTS?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Build
+        run: | 
+          hugo --minify
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./exampleSite/public  # defaults to ./public # directorio source para hacer el deploy (debe coincidir con lo configurado para hacer el build en config.toml), eg hace el Deploy desde ./docs
```
Comment (ryanfox1985) [inline .github/workflows/main.yml:34]: could you translate this spanish text to english?

**Review by ryanfox1985 — CHANGES_REQUESTED** [review]

**Review by ryanfox1985 — APPROVED** [review]

**Comment by chepo92** [comment]
@ryanfox1985  can you run the deploy again? IDK why it failed, it worked ok in my fork, maybe is it blocked to push to gh-pages. 

```
  /usr/bin/git push origin gh-pages
  remote: fatal error in commit_refs        
  To https://github.com/devcows/hugo-universal-theme.git
   ! [remote rejected] gh-pages -> gh-pages (failure)
  error: failed to push some refs to 'https://github.com/devcows/hugo-universal-theme.git'
  Error: Action failed with "The process '/usr/bin/git' failed with exit code 1"
```

**Comment by ryanfox1985** [comment]
Hello @chepo92, thanks for the feedback, I noticed too. I think is a matter of repository configuration. I'll take a look.

**Comment by ryanfox1985** [comment]
image

**Comment by ryanfox1985** [comment]
I found the issue, I created manually the gh-pages branch and I rerun the flow https://github.com/devcows/hugo-universal-theme/runs/3208399294.

**Comment by chepo92** [comment]
Nice, now you also may want to add the url to the sidebar/description of the repo

**Comment by ryanfox1985** [comment]
@chepo92 I added to the description https://github.com/devcows/hugo-universal-theme#universal-theme-for-hugo . Do you mean this right?

**Comment by chepo92** [comment]
I mean this, you may also add the tag/topics (eg "hugo" "template") to make it easier to find this repo: 
ss

**Comment by ryanfox1985** [comment]
nice catch, I updated. Thanks!

**Pull request merged**